### DISCUSSION TITLE
User Logos in multiplayer

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#19031] [Feature requests]
Right now when someone joins a server, they could impersonate another user. I propose we allow a small icon next to the user's name so people can have a unique logo. It is not a perfect solution but an improvement over the current system.

**Comment by ocalhoun6** [comment]
It would still be trivially easy to impersonate someone -- you'd just need to take a screenshot of their icon first.

**Comment by 733737** [comment]
I still have wanted this feature for months, would be a cool option.

**Comment by ageekhere** [comment]
Related, though has not been updated since 2016
https://github.com/OpenRCT2/OpenRCT2/projects/3
